test/updateIn-spec.js

```diff
@@ -27,6 +27,12 @@ describe('u.updateIn', () => {
     expect(result).to.eql({ a: [0, 3, 0] });
   });
 
+  it('can create array if all keys are numbers', () => {
+    const result = u.updateIn('a.1', 3, null);
+
+    expect(result).to.eql({ a: [, 3] }); // eslint-disable-line no-sparse-arrays
```

Comment:
well, you'd also need to test that it is an array if you did that. You could also just use the 0 index. I'm good w/ any of these